.github/workflows/robustness.yaml

```diff
@@ -10,3 +10,4 @@ jobs:
       count: 12
       testTimeout: 30m
       artifactName: main
+      runs-on: ubuntu-latest-8-cores
```

Comment:
Guessing
```suggestion
      runs-on: "'ubuntu-latest-8-cores'"
```

Reply:
true 😂  maybe we should use array to it. let me try it